### DISCUSSION TITLE
想hook一下android14的 pthread_create 函数，在libart.so 里边没有找到对应的函数，有大佬给解释一下吗 ^_^

### DISCUSSION THREAD

**Discussion by 20083017** [bytedance/android-inline-hook#51] [Q&A]
![image](https://github.com/bytedance/android-inline-hook/assets/8308226/8c18c540-2f6c-4c78-a60f-c199a20068d3)

**Comment by Colibrow** [comment]
在libc.so里面看看呢

**Reply by 20083017** [reply]
https://tech.qimao.com/xian-cheng-shu-liang-you-hua/  我是参照这篇文章搞的，暂时还没想明白文章里边为啥说是hook libart.so

而且直接hook libc.so 的pthread_create 好像不行，这个会死循环好像 ~

**Reply by Colibrow** [reply]
plthook建议使用https://github.com/bytedance/bhook
